src/App.js

```diff
@@ -1,19 +1,55 @@
-import React from 'react';
```

Comment:
Overall, you did a great job.  Code is clean, readable, and you coded the React components well.  Nice work!